features.js

```diff
@@ -851,20 +857,26 @@ module.exports = {
 
     // check whether audio is received after 10 seconds
     receivingaudio10s: function(client, peerConnectionLog) {
-        var count = 0;
         var receivedAudio = undefined;
+        var timeConnected = false;
         for (var i = 0; i < peerConnectionLog.length; i++) {
-            if (peerConnectionLog[i].type === 'getStats') {
-                if (count++ === 10) {
-                    Object.keys(peerConnectionLog[i].value).forEach(id => {
-                        var report = peerConnectionLog[i].value[id];
-                        if (report.type === 'ssrc' && (report.kind === 'audio' || report.mediaType === 'audio') && id.indexOf('_recv')) {
-                            receivedAudio = {
-                                packetsReceived: report.packetsReceived,
-                                jitterbufferms: report.googJitterBufferMs
-                            };
-                        }
-                    });
+            if (peerConnectionLog[i].type === 'oniceconnectionstatechange' && !timeConnected) {
```

Comment:
These lines look very similar to th eones for video, a refactor could be a good idea while you change it

Reply:
this is copy-paste with different variable names for the purpose of readability...